Maintenance Announcement EU (#2031)
galaxyproject/galaxy-hub: content/bare/eu/usegalaxy/notices.md

</div-->

<div class="alert" style="background: #fd8c00;">

> ### **Database maintenance on 12 June**

Reviewer: ```suggestion
### **Galaxy's database server maintenance on 12 June**
```

> Maintenance will take place on 12 June from 9:30am CEST (7:30am GMT) and should not exceed 5:30pm CEST (3:30pm GMT). usegalaxy.eu will not be reachable during this period. You do not need to take any action.  

Reviewer: ```suggestion
Maintenance will take place on 12 June from 9:30 am CEST (7:30 am GMT) until 5:30 pm CEST (3:30 pm GMT). usegalaxy.eu will not be reachable during this period. You do not need to take any action.  
```

> <div class="alert" style="background: #fd8c00;">
> ### **Database maintenance on 12 June**
> 
> Maintenance will take place on 12 June from 9:30am CEST (7:30am GMT) and should not exceed 5:30pm CEST (3:30pm GMT). usegalaxy.eu will not be reachable during this period. You do not need to take any action.  
> [Check out our blog post for more information!](https://galaxyproject.org/news/2023-06-06-eu-maintenance)
> 

Reviewer 2: ```suggestion
<div class="alert" style="background: #fd8c00;">

#### **Database maintenance on 12 June**

Maintenance will take place on 12 June from 9:30am CEST (7:30am GMT) and should not exceed 5:30pm CEST (3:30pm GMT). usegalaxy.eu will not be reachable during this period. You do not need to take any action.  
[Check out our blog post for more information!](https://galaxyproject.org/news/2023-06-06-eu-maintenance)

</div>

```

![grafik](https://github.com/galaxyproject/galaxy-hub/assets/43052541/32927cfe-cc35-4b65-ab91-2869b62532ae)
